fixtures/fixture.py

```diff
@@ -69,19 +70,36 @@ def setup_test_repo():
     origin.pull('assemblies-2')
 
     logging.info("Installing the Pantheon uploader script..")
-    try:
-        subprocess.check_call(
-            "curl -o pantheon.py https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/pantheon.py",
-            shell=True)
-    except subprocess.CalledProcessError as e:
-        logging.error("Unable to install the uploader script")
-        raise e
+    # try:
+    #     subprocess.check_call(
+    #         "curl -o pantheon.py https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/pantheon.py",
+    #         shell=True)
+    # except subprocess.CalledProcessError as e:
+    #     logging.error("Unable to install the uploader script")
+    #     raise e
+    #
+    # os.chdir(project_dir_git)
+    uploader_path = os.path.join(os.getcwd(), 'uploader-repo')
+    print(uploader_path)
+
+    if os.path.isdir(uploader_path):
+        shutil.rmtree(uploader_path)
 
-    os.chdir(project_dir_git)
+    os.mkdir(uploader_path)
+
+    try:
+        os.chdir(uploader_path)
+        git.Repo.clone_from(uploader_repo, uploader_path)
+        print(os.system("ls"))
+        os.system("chmod +x install.sh")
+        os.system("./install.sh")
```

Comment:
This step is likely to fail on jenkins CI setup as there are pip install steps in the install.sh which would not be successful as we cannot do a pip install on jenkins node without a virtual env.